### DISCUSSION TITLE
Unable to find docker-buildx-plugin 0.10.5 for bullseye

### DISCUSSION THREAD

**Discussion by Xzelsius** [docker/buildx#1843] [Q&A] [answered]
I am trying to build a custom image containing docker cli, buildx and compose.
But the build currently fails with

```
#10 2.808 E: Version '0.10.5-1~debian.11~bullseye' for 'docker-buildx-plugin' was not found 
```

When I verify this manually I can't find the 0.10.5 package for bullseye either
```
docker-buildx-plugin | 0.10.4-1~debian.11~bullseye | https://download.docker.com/linux/debian bullseye/stable amd64 Packages
docker-buildx-plugin | 0.10.2-1~debian.11~bullseye | https://download.docker.com/linux/debian bullseye/stable amd64 Packages
```

Did something happen with the release of 0.10.5 or do I need to wait a bit till it is published for bullseye?
(release of [v0.10.5](https://github.com/docker/buildx/releases/tag/v0.10.5) was 3 days ago)

See this PR https://github.com/swissgrc/docker-azure-pipelines-dockercli/pull/5 for the full Dockerfile.

**Comment by crazy-max** [comment]
Buildx 0.10.5 pkg update has been merged https://github.com/docker/docker-ce-packaging/pull/900 but not yet published on download.docker.com: https://download.docker.com/linux/debian/dists/bullseye/pool/stable/amd64/

I guess Renovate relies on GitHub Releases to create the PR but what is published in GitHub Release in this repo might not be yet available in Docker packages. cc @thaJeztah

**Reply by thaJeztah** [reply, marked as answer]
Yes, releases on GitHub are created when the project tagged a release (which currently also provides binary downloads on GitHub). Packages on download.docker.com, are part of Docker's release process, which (currently) happens together with (patch)releases of the Docker Engine, so sometimes there may be some latency, and buildx plugin packages will appear when the next engine release is done.

We're working on separating those releases from the Docker Engine releases, but this still requires some amount of work to be completed.


On a bright note: the Docker Engine 24.0.2 release is currently rolling, and packages for buildx 0.10.5 should be available ... very soon (within an hour or so 🤞)

**Reply by Xzelsius** [reply]
Thank you for your detailed replies!
Then I'll wait a little longer :)

**Reply by thaJeztah** [reply]
Release pipeline finished; https://github.com/moby/moby/releases/tag/v24.0.2

Packages should be available now; https://download.docker.com/linux/debian/dists/bullseye/pool/stable/amd64/

<img width="984" alt="Screenshot 2023-05-26 at 11 17 58" src="https://github.com/docker/buildx/assets/1804568/8bde0e97-db93-4e13-aadc-b04b44b410ee">
